packages/xstate-wallet/src/store/state-utils.ts

```diff
@@ -194,7 +194,7 @@ export function convertToNitroOutcome(outcome: Outcome): NitroOutcome {
       ];
     case 'MixedAllocation':
       // FIXME: is this a typo?
```

Comment:
?

Reply:
The code is literally broken, but there was an unused var that I removed

Reply:
Ok. If you have any more context / understanding of this comment, could you improve it? The comment is a question, but it sounds like you are sure that this is broken. 

Appreciate that this is outside scope of your PR, but it seems like a good opportunity.